tests/RobotFramework/libraries/ThinEdgeIO/ThinEdgeIO.py

```diff
@@ -1015,20 +1015,22 @@ def register_entity(
 
     @keyword("List Entities")
     def list_entities(
-            self, device_name: str = None 
+            self, params: Dict[str, str] = None, device_name: str = None
```

Comment:
I don't think the putting all of the query parameter mappings in the params is a good idea here (it just makes it a lot harder to use).

```suggestion
            self, entity_type: Optional[str, str] = None, parent: Optional[str, str] = None, root: Optional[str, str] = None, device_name: str = None
```

And if people still want to use a dictionary, then we could just add kwargs support to the function signature, and then people can pass this stuff as a dictionary (as that is more python like)